rfcs/0045-deprecate-url-syntax.md

```diff
@@ -0,0 +1,70 @@
+---
+feature: deprecate_url_syntax
+start-date: 2019-04-28
+author: Michael Raskin
+co-authors: 
+related-issues: 
```

Comment:
https://github.com/NixOS/nixpkgs/pull/27809
https://github.com/NixOS/nixpkgs/pull/46378

Reply:
I am not sure: the RFC template seems to imply this is for PRs implementing the RFC, not for the changes the RFC intends to undo.